pkg/backup/client.go

```diff
@@ -226,6 +227,20 @@ func BuildBackupRangeAndSchema(
 				return nil, nil, errors.Trace(err)
 			}
 			tableInfo.AutoIncID = globalAutoID
+
+			if tableInfo.PKIsHandle && tableInfo.ContainsAutoRandomBits() {
```

Comment:
must it to meet both requirements?

Reply:
yes, auto_random must be assigned on PKIsHandle table.